### ISSUE TITLE
[help wanted] Support more models trained with CTC loss

### ISSUE THREAD

**Issue by csukuangfj** [k2-fsa/sherpa#203]
The following page lists all supported CTC models currently we have in sherpa
https://k2-fsa.github.io/sherpa/cpp/pretrained_models/offline_ctc.html

Namely, models from icefall, wenet, and torchaudio (wav2vec 2.0) are supported.

---

Basically, sherpa can support any models trained using CTC loss. If you have such a model and are able to export it via torchscript,
it is possible to support it in sherpa in a straightforward way.

Please let us know if you have such a model and provide the link to the torchscript model.
